src/components/Swap/ReverseButton.tsx

```diff
@@ -17,7 +17,7 @@ const StyledReverseButton = styled(Button)`
   justify-content: center;
   left: 50%;
   position: absolute;
-  transform: translate(-50%, -60%);
+  transform: translate(-50%, calc(-50% - 2px));
```

Comment:
Comment on why the `-2px` are necessary, so that they're not inadvertently removed in the future.

Reply:
done